lampo-bdk-wallet/src/lib.rs

```diff
@@ -149,9 +149,12 @@ impl WalletManager for BDKWalletManager {
             .lock()
             .unwrap()
             .get_address(bdk::wallet::AddressIndex::New);
-        Ok(NewAddress {
-            address: address.address.to_string(),
-        })
+        match address{
+            Ok(info) => Ok(NewAddress {
+                address: format!("{:?}", info.payload), 
```

Comment:
`{:?}` is a debug format, so this is wrong because we are sending this info to the user